### PR TITLE
Updates to v3.1, adds download check

### PR DESCRIPTION
**Pull request by Phoebe-B**

### CODE REVIEW

File: CHANGELOG.MD
Comment (Phoebe-B) [inline CHANGELOG.MD:1]: Change to correct date before merging

**Review by superdevx64 — APPROVED** [review]

File: CHANGELOG.MD
Comment (olasitarska) [reply CHANGELOG.MD:1]: On versioning - is this actually a major version release? The reason I'm asking is because it doesn't require any change from the integrators to adopt it.

File: tests/test_applicants.py
Comment (olasitarska) [inline tests/test_applicants.py:21]: Maybe we should introduce some constant variable for this? Happy for this to be considered as a future improvement and not blocking this change.

**Review by olasitarska — APPROVED** [review]

File: tests/test_applicants.py
Comment (superdevx64) [reply tests/test_applicants.py:21]: Agreed on both counts, will make a ticket

File: CHANGELOG.MD
Comment (Phoebe-B) [reply CHANGELOG.MD]: Have updated to minor version release

File: tests/test_webhook_verification.py
Comment (superdevx64) [inline tests/test_webhook_verification.py:13]: @Phoebe-B I realise in hindsight this should have v3 in the href, since it's the webhook version, not the API version

File: tests/test_webhook_verification.py
Comment (superdevx64) [inline tests/test_webhook_verification.py:23]: @Phoebe-B Same comment as above:

I realise in hindsight this should have v3 in the href, since it's the webhook version, not the API version

File: onfido/exceptions.py
Comment (ciaran16) [inline onfido/exceptions.py:26]: Minor: I wouldn't subclass `OnfidoError` here in case clients have error handling for it. You could probably just subclass Exception, as it's not related to errors making a request like I think the others are.

Comment (superdevx64) [reply onfido/exceptions.py]: Thanks, good point, have changed

**Review by kulak-at — APPROVED** [review]

**Pull request merged**
